### PR TITLE
Add hvbc.club

### PR DESCRIPTION
**Pull request by frivoxfr**
### General:
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*
* [x] The pull request title is descriptive. *(ex. `Added GoldenPvP Network`, not `Updated metadata.json`)*

### Mapping Additions or Updates:
* [x] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [x] No changes were made to the file's formatting (4 spaces per tab).
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [x] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/CheatBreakerNet/Client-API/blob/master/mappings/metadata.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string, which should match the folder name *(ex. `goldenpvpnetwork`)*
  - [x] `name`: a string *(ex. `GoldenPvP Network`)*
  - [x] `description`: hook between 16 and 40 characters *(ex. `We're the home of classic PvP gamemodes`)* 
  - [x] `addresses`: an array with lowercase strings *(ex. `["goldenpvp.net", "routing.center"]`)*
    - You do not need to specify subdomains, CheatBreaker services automatically detect them.
  - [x] `primaryAddress`: the primary address that people connect to with (please include the subdomain if required) *(ex. `mc.goldenpvp.net`)*
  - [x] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.7.*", "1.8.9"]` - Must be versions or subversions supported by CheatBreaker)*
  - [x] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.7.10` - Must be a subversion supported by CheatBreaker)*
  - [x] `primaryColor`: a hexademical color code that primarily distinguishes the server *(ex. `#00FFFF`)* 
  - [x] `secondaryColor`: a hexademical color code that accompanies the `primaryColor` of the server *(ex. `#FF0000`)*
  - [x] `primaryRegion`: the primary region where your server operates in *(ex. `EU`)*
  - [x] `regions`: a list of regions where you have servers located that service your players *(ex. `["EU", "NA", "AS"]`)*
  - [x] `gameTypes`: a list of games that describe the content on your server, must be a max of 3 listed *(ez. `["PVP", "UHC", "HCF"]`)*
  - [ ] (optional) `website`: url of server website, must include URL schema (http:// or https://) *(ex. `https://www.goldenpvp.net`)*
  - [ ] (optional) `store`: url of server store, must include URL schema (http:// or https://) *(ex. `https://store.goldenpvp.net`)*

### Socials
* [ ] (optional) `twitter`: username of twitter account without the @ *(ex. GoldenPvPNet)*
* [x] (optional) `discord`: invite link to discord *(ex. https://discord.com/invite/3EAzBxf)*
* [ ] (optional) `youtube`: slug or username of youtube channel *(ex. GoldenPvP)*
* [ ] (optional) `instagram`: username of instagram account *(ex. GoldenPvPNet)*
* [ ] (optional) `twitch`: username of twitch account *(ex. GoldenPvPNet)*
* [ ] (optional) `telegram`: slug of telegram group *(ex. GoldenPvP-Network)*
* [ ] (optional) `reddit`: slug of subdreddit wtih 'r/' *(ex. GoldenPvP)*
* [ ] (optional) `tiktok`: username of tiktok account *(ex. GoldenPvP)*
* [ ] (optional) `facebook`: slug of facebook page *(ex. GoldenPvP)*

#### Icon
* [x] My icon is a `png` file.
* [x] I have uploaded my icon to my server folder *(ex. `goldenpvpnetwork`)* and named it `icon.png`.
* [x] My icon has a transparent background and is square (1:1 aspect ratio).
* [x] My icon is `64` pixels in width and height.
* [x] My icon is my own property and complies with relevant copyright/privacy laws.

#### Logo
* [x] My logo is a `png` file.
* [x] I have uploaded my logo to my server folder *(ex. `goldenpvpnetwork`)* and named it `logo.png`.
* [x] My logo has a transparent background and is square (1:1 aspect ratio).
* [x] My logo is `108` pixels in width and height.
* [x] My logo is my own property and complies with relevant copyright/privacy laws.

#### Discord Logo
* [x] My logo is a `png` file.
* [x] I have uploaded my logo to my server folder *(ex. `goldenpvpnetwork`)* and named it `discord.png`.
* [x] My logo has a transparent background and is square (1:1 aspect ratio).
* [x] My logo is between `512` and `1024` pixels in width and height.
* [x] My logo is my own property and complies with relevant copyright/privacy laws.

#### Primary Background (Only required for partnered servers)
* [ ] My primary background is a `png` file.
* [ ] I have uploaded my primary background to my server folder *(ex. `goldenpvpnetwork`)* and named it `primarybackground.png`.
* [ ] My primary background is `774` pixels in width and `363` pixels in height.
* [ ] My primary background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.
* [ ] My primary background doesn't contain any markings, text, or logos (unless part of a build).

#### Secondary Background (Only required for partnered servers)
* [ ] My secondary background is a `png` file.
* [ ] I have uploaded my secondary background to my server folder *(ex. `goldenpvpnetwork`)* and named it `secondarybackground.png`.
* [ ] My secondary background is `447` pixels in width and `172` pixels in height.
* [ ] My secondary background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.
* [ ] My secondary background doesn't contain any markings, text, or logos (unless part of a build).

### CODE REVIEW

File: mappings/servers/hvbc/metadata.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "id": "hvbc",
+    "name": "hvbc",
```
Comment (ShonP40) [inline mappings/servers/hvbc/metadata.json:3]: Please use the actual server name here.
I'm guessing that it should be `HVBC` in your case (capitalized)
Unless you have a full server name

File: mappings/servers/hvbc/metadata.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "description": "Practice Server For Legit player",
+    "addresses": [
+        "hvbc.club",
+        "na.hvbc.club",
```
Comment (ShonP40) [inline mappings/servers/hvbc/metadata.json:7]: As stated in the readme, there is no point in specifying subdomains here as CB already matches them automatically

File: mappings/servers/hvbc/metadata.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        "dev.hvbc.club",
```
Comment (ShonP40) [inline mappings/servers/hvbc/metadata.json:8]: same as above

**Review by ShonP40 — CHANGES_REQUESTED** [review]

**Comment by frivoxfr** [comment]
I It should be fine now

File: mappings/servers/hvbc/metadata.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+    "id": "hvbc",
+    "name": "HVBC",
+    "description": "Practice Server For Legit player",
+    "addresses": [
+        "hvbc.club",
+    ],
+    "primaryAddress": "hvbc.club",
```
Comment (ShonP40) [inline mappings/servers/hvbc/metadata.json:8]: I've noticed that this address does not respond to MC pings but `na.hvbc.club` does

So I'm guessing the the `na.` address is your primary address, and it is the one you should use here

Leave the `addresses` section as is

File: mappings/servers/hvbc/metadata.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
+{
+    "id": "hvbc",
+    "name": "HVBC",
+    "description": "Practice Server For Legit player",
```
Comment (Tellinq) [inline mappings/servers/hvbc/metadata.json:4]: If you are concerned about proper grammar (which we do care to a degree), I recommend that you adjust the description.

A more proper description could be:
- “A practice server for legit players”
- “A practice server for legitimate players”

You don’t need to be restricted to these two options, but these should suffice while keeping the main description value.

If you want to change the description entirely, feel free to do so

**Comment by frivoxfr** [comment]
I made the changes

File: mappings/servers/hvbc/metadata.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "hvbc",
     "name": "HVBC",
-    "description": "Practice Server For Legit player",
+    "description": "Practice Server For Legitimate player",
```
Comment (Tellinq) [inline mappings/servers/hvbc/metadata.json:4]: A few more grammar tweaks I recommend

Descriptions should not be title case (“Practice Server for Legitimate Player” -> “Practice server for legitimate player”)

pretty sure “player” seems to be intended as a plural use. I would just use “players”

in descriptions, typically you don’t want an abrupt start. That’s why I had “A” at the start

**Comment by frivoxfr** [comment]
it's good

File: mappings/servers/hvbc/metadata.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
+{
+    "id": "hvbc",
+    "name": "HVBC",
+    "description": "A practice server for legitimate players",
```
Comment (ShonP40) [inline mappings/servers/hvbc/metadata.json:4]: You went over the 40-char limit

You should replace `legitimate` with `legit` probably

**Review by ShonP40 — CHANGES_REQUESTED** [review]

**Comment by frivoxfr** [comment]
it's good

**Review by ShonP40 — APPROVED** [review]

File: mappings/servers/hvbc/metadata.json [hunk 8]
```diff
@@ -0,0 +1,30 @@
+{
+    "id": "hvbc",
+    "name": "HVBC",
+    "description": "A practice server for legit players",
+    "addresses": [
+        "hvbc.club",
```
Comment (ShonP40) [inline mappings/servers/hvbc/metadata.json:6]: delete this `,`

**Comment by frivoxfr** [comment]
yep

**Review by ShonP40 — APPROVED** [review]

**Review by Tellinq — APPROVED** [review]
LGTM

**Review by Tellinq — APPROVED** [review]
LGTM

File: mappings/servers/hvbc/metadata.json [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    ],
+    "primaryRegion": "NA",
+    "regions": [
+        "NA"
+    ],
+    "gameTypes": [
+        "PvP"
+    ],
+    "socials": {
+        "discord": "https://discord.gg/hvbc",
```
Comment (ShonP40) [inline mappings/servers/hvbc/metadata.json:28]: same here (remove the `,`)

**Review by ShonP40 — APPROVED** [review]

**Review by Tellinq — APPROVED** [review]
Good to me if the automated action succeeds

**Pull request merged**